style.css

```diff
@@ -15,69 +15,67 @@
   align-items: center;
 }
 
-  #logo {
-    text-decoration: none;
-    font-family: 'Roboto', sans-serif;
-    font-size: 18px;
-    font-weight: 700;
-    color: #6070FF;
-    letter-spacing: 0.03em;
-    line-height: 20px;
-    font-style: normal;
-    margin-right: 5px;
-  }
-
-
+#logo {
+  text-decoration: none;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+  line-height: 20px;
+  font-style: normal;
+  margin-right: 5px;
+}
 
-  body {
-    height: 100vh;
-    background-image: url(./img/shapes.png);
-    background-size: cover;
-    background-repeat: no-repeat;
-    justify-content: center;
-    border-radius: 0 0 0 15px;
-    color: #172B4D;
-    font-family: "poppins", sans-serif;
-  }
+body {
+  height: 100vh;
```

Comment:
- [OPTIONAL] Setting the height of the body to "100vh" may cause issues and a bad experience when a user tries to scroll down. Kindly remove it.